### PR TITLE
ip: return all errors to main

### PR DESCRIPTION
**Pull request by rminnich**
Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

**Comment by rminnich** [comment]
this is a slightly different take on returning all errors to main, and only having man log.Fatalf. 

It should now be simpler to write tests. The next step I suppose is to make a Cursor interface which contains command info now held in globals.

File: cmds/ip/ops.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
```
Comment (rjoleary) [inline cmds/ip/ops.go]: 2018

Comment (rminnich) [reply cmds/ip/ops.go]: done

Comment (rjoleary) [reply cmds/ip/ops.go]: It still says 2017... I'll automatically change it later with the script.

**Pull request merged**